index.js

```diff
@@ -40,9 +47,29 @@ const Web3 = require('web3');
   let web3 = new Web3(engine);
   web3.eth.defaultAccount = hubotWalletAddress;
 
-  let contracts = kreditsContracts(web3, config);
+  //
+  // Instantiate contracts
+  //
+  let contractConfig = {};
+  if (process.env.KREDITS_CONTRACT_ADDRESS) {
+    contractConfig = { Kredits: { address: process.env.KREDITS_CONTRACT_ADDRESS }};
+  }
+  let contracts = kreditsContracts(web3, contractConfig);
   let kredits = contracts['Kredits'];
 
+  //
+  // Instantiate IPFS API client
+  //
+  let ipfsConfig = {};
+  if (process.env.IPFS_API_HOST) {
+    ipfsConfig = {
+      host: process.env.IPFS_API_HOST,
+      port: process.env.IPFS_API_PORT,
+      protocol: process.env.IPFS_API_PROTOCOL
```

Comment:
maybe it would be nice to have defaults here? for example for the protocol to be https if not explicitly specified